src/setupTests.js

```diff
@@ -5,3 +5,8 @@
 import "@testing-library/jest-dom/extend-expect"
 
 jest.mock("react-ga")
+
+beforeAll(() => {
```

Comment:
This has been added to remove the below mentioned warning.

`error: not implemented: window.computedstyle(elt, pseudoelt)`

[https://github.com/nickcolley/jest-axe/issues/147](url)